app/scripts/modules/kubernetes/src/v2/manifest/showManifestDetails.component.ts

```diff
@@ -0,0 +1,81 @@
+import { module, IComponentOptions, IController } from 'angular';
+import { trim } from 'lodash';
+import { ReactInjector } from 'core/reactShims';
+
+function stripQuotes(str: string): string {
+  return trim(str, '"');
+}
+
+function extractAnnotations(annotations?: any): any {
+  if (annotations == null) {
```

Comment:
Not sure if it's possible that this condition could be `true`, but if it was, it looks like you would still render the link, but it wouldn't work.

Reply:
i've disabled the link when annotations are not present in the metadata and i've removed this condition.